product_docs/docs/pem/9/managing_certificates_copy.mdx

```diff
@@ -0,0 +1,283 @@
+---
+title: "Configuring Secure Connection with PEM Server"
+navTitle: "Certificates Copy"
+legacyRedirectsGenerated:
+  # This list is generated by a script. If you need add entries, use the `legacyRedirects` key.
+  - "/edb-docs/d/edb-postgres-enterprise-manager/user-guides/administrators-guide/8.0/managing_certificates.html"
+# This file is moved from pem_admin/04_managing_certificates
+redirects:
+- /pem/latest/pem_admin/04_managing_certificates/
+---
+PEM, by default, supports secured SSL or TLS connectivity and acts as its own certificate authority (CA) to generate certificates and keys for the PEM server and agent. 
+
+During the initial PEM installation, the installer generates its own SSL CA certificates and keys for the PEM server and agent, which are then used for an encrypted connection between them. You can also replace the PEM self-signed CA certificate and key with trusted third-party CA certificates when configuring SSL on the PEM server. 
```

Comment:
The SSL CA certificates and keys for the PEM server and agent gets generated during the PEM installation. These certificates and keys encrypt the connections between them. You can replace the PEM self-signed CA certificate and key with trusted third-party CA certificates.

Reply:
Done